lib/smart_proxy_container_gateway/container_gateway_main.rb

```diff
@@ -4,11 +4,13 @@
 require 'sequel'
 module Proxy
   module ContainerGateway
+    extend Proxy::DHCP::DependencyInjection
```

Comment:
Clearly this needs to be something else than DHCP, so pretty much a copy of https://github.com/theforeman/smart-proxy/blob/develop/modules/dhcp/dependency_injection.rb